storagenode/storagenodedb/uuid.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+)
+
+// bytesToUUID is used to convert []byte to UUID
+func bytesToUUID(data []byte) (uuid.UUID, error) {
```

Comment:
this method is used in satellitedb and in storagenodedb. What if we place it in pkg or private?

Reply:
let me do it in another PR since it's not in scope of this one

Reply:
I feel like this should be done within the PR, else its forgotten about.
Especially if reusing existing code, moving it into another package is fine.

Reply:
if i do it within this PR it's size would become really big, cause i'll have to update a lot of repositories in satellite and storagenode. I'll create a ticket in Jira not to forget about it